.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions/setup-go@v2.1.3
+        with:
+          go-version: '1.16'
+      - name: Build
+        run: ./goyek.sh -v -ci
```

Comment:
Can this be put into a Makefile target to allow users to run this locally?

Reply:
I just decided to create a default `goyek` make target that executes `./goyek.sh`. Passing flags via Make is not fun 😉